active/community.md

```diff
@@ -0,0 +1,86 @@
+# Online Community Working Group
```

Comment:
It’d be nice to update the PR title to match this, or the other way around. As-is the proposal reads to me like a focused "online platforms working group", while I was expecting a broad "community working group" (for example including in-person events like Django Social).

Reply:
Done. I think offline events would be better served by either an events WG or a conference WG

Reply:
I would update the filename also 👍